tools/report-converter/tests/unit/analyzers/jscpd_output_test_files/jscpd-report.json

```diff
@@ -0,0 +1,133 @@
+{
```

Comment:
In the test file we should check the generated plist and check its content. Currently the test doesn't use the report converter.